### DISCUSSION TITLE
제3회 콘테스트 개최 - 공유 콘텐츠 편

### DISCUSSION THREAD

**Discussion by eun-byeol** [woowacourse-teams/2024-ody#364] [Ideas]
앱에 참여하지 않아도 공유하면 재미있을 것 같은 콘텐츠 올리기 + 디자인(와이어프레임) 포함

보상: 칭찬

**Comment by coli-geonwoo** [comment]
https://sly-face-106.notion.site/46831135088b4db096497801ad880c03?pvs=74



1)
![image (2)](https://github.com/user-attachments/assets/49de405a-4df2-41f7-abba-48f29ad6004f)
![image (3)](https://github.com/user-attachments/assets/1db26a61-8f53-4bae-9689-b7daf620a57b)
![image](https://github.com/user-attachments/assets/5bb7db01-82ca-4e6c-9bf1-b64f7f645368)
![image (1)](https://github.com/user-attachments/assets/35389abf-f3dd-4b16-94e8-bf8d630f4294)

**Reply by coli-geonwoo** [reply]
1. 지각 재촉기능 : 나갔니

**Reply by coli-geonwoo** [reply]
2. 지각 핑계 뱃지 공유

**Comment by hyeon0208** [comment]
![제목_없는_아트워크](https://github.com/user-attachments/assets/ae4c599f-7e33-4c7a-9e9c-24bc6c485b20)

**Comment by mzeong** [comment]
<img width="1003" alt="스크린샷 2024-08-13 오후 5 22 07" src="https://github.com/user-attachments/assets/907a66b4-8ea1-4fd7-9483-da9f86529a39">
<img width="856" alt="스크린샷 2024-08-13 오후 5 22 54" src="https://github.com/user-attachments/assets/2eaa95d5-cdd8-44c0-a4fa-3bd6b250ad48">

**Reply by eun-byeol** [reply]
납득/납득 불가 투표 기능 너무 좋은데요? 재밌어요 짱짱

**Reply by mzeong** [reply]
마피아 게임 지각자 ver

**Reply by mzeong** [reply]
오늘의 지각왕

**Comment by kimhm0728** [comment]
## **앱이 설치되지 않은 사용자에게도 엔터적 요소를 줄 수 있는 컨텐츠**

### 1 
자신의 위치 현황을 공유한다.
안드로이드를 사용하지 않는, 어플을 설치하지 않은 사람에게도 공유할 수 있음.
→ 약속을 잡은 모든 친구들이 오디를 사용하지 않아도 일부 기능을 활용할 수 있을 것 같다.

```
"OOO님이 10분 후 도착 예정이에요! 서둘러 주세요!"
"OOO님이 지각 위기예요 😱"
"긴급 속보: 나 아직 집이야! 조금만 늦어도 화내지 않기 약속!"
"현재 위치: 집. 약속 시간까지 10분. 이거 어떻게든 해볼게, 하지만...!"
"내가 또 늦을 것 같아... 조금만 기다려 줄 수 있지?"
```
<img width="438" alt="스크린샷 2024-08-13 오후 5 20 47" src="https://github.com/user-attachments/assets/4c318bc4-bb77-4f28-b082-2584f41bfe34">


### 2

지각 횟수, 지각하지 않은 횟수에 따라 뱃지를 부여한다.
뱃지를 다른 사람들에게 공유할 수 있는 기능.

```
"OOO님은 지각을 총 3번 하셨어요. 몇 번이나 지각했는지 체크해보세요!""
"OOO님은 `게으름뱅이`에요. 지각러인지 정시러인지 확인해보세요!"
```

<img width="628" alt="스크린샷 2024-08-13 오후 5 21 20" src="https://github.com/user-attachments/assets/40e143cf-ee78-4787-93fa-857bfbdbfbe6">

**Reply by kimhm0728** [reply]
1

**Reply by kimhm0728** [reply]
2

**Comment by haeum808** [comment]
<img width="526" alt="image" src="https://github.com/user-attachments/assets/c1f8bef8-1150-4137-9ba8-ef7568865688">

![이름 없는 노트북](https://github.com/user-attachments/assets/818e18dd-ef1d-4d48-a5bc-28c205ec9d77)

**Comment by eun-byeol** [comment]
# 약속 시간 전 공유 콘텐츠
## 1. 정시 도착 예정 인증서 발급
나 정시에 도착함 ㅇㅈ?
![image](https://github.com/user-attachments/assets/ef0170b2-cb62-4be8-b967-c49c91811a92)

## 2. 개근상 발급 - 꾸준히 정시러를 실천하는 사람 대상
나 진짜 안 늦었다 ㅇㅈ?
![image](https://github.com/user-attachments/assets/20dc3361-9c93-4c1f-8b37-fa2b075933e8)

## 3. 지각 반성문
나 지각할 것 같아,,, 자수하고 선처받기
![image](https://github.com/user-attachments/assets/4ef1b6cf-9151-4018-afd3-223b69e12e55)

# 약속 시간 후 공유 콘텐츠
## 1. 지각러, 정시러 결과 공유
정시 자각러 박제
![image](https://github.com/user-attachments/assets/14807fea-6e48-4364-9599-e3a4f591b876)

**Reply by eun-byeol** [reply]
1. 정시 도착 예정 인증서 발급

**Reply by eun-byeol** [reply]
2. 개근상 발급 - 꾸준히 정시러를 실천하는 사람 대상

**Reply by eun-byeol** [reply]
3. 지각 반성문

**Reply by eun-byeol** [reply]
4. 지각러, 정시러 결과 공유 - 레이싱

**Comment by aprilgom** [comment]
https://www.figma.com/design/ODq9jq2IesLozVuNFjJblA/Untitled?node-id=0-1&t=EpPrn61KcmN60Quv-1
<img width="472" alt="스크린샷 2024-08-13 17 26 30" src="https://github.com/user-attachments/assets/02c18183-49d2-4190-91e3-56615dc215b1">
날씨에 따른 재촉
